files/config/default/etc/workflows/uct-include-autotrim-detection.xml

```diff
@@ -58,7 +75,7 @@
       max-attempts="3"
       retry-strategy="retry"
       fail-on-error="true"
-      if="${trim_detect_media} AND ${trim_detect_audio} AND ${audio_trim_ishour}"
+      if="${trim_detect_media} AND ${trim_detect_audio} AND ${audio_trim_ishour} AND NOT(${empty_venue_detect})"
```

Comment:
empty_venue_detect sounds like it's an action (true/false to do something). As it holds the result, just make it empty_venue (true if the venue was detected as empty).